src/coprocessor/tracker.rs

```diff
@@ -434,6 +434,36 @@ impl<E: Engine> Drop for Tracker<E> {
         if let TrackerState::ItemFinished(_) = self.current_stage {
             self.on_finish_all_items();
         }
+
+        if self.current_stage != TrackerState::AllItemFinished
```

Comment:
will it log twice?

Reply:
slow log depend on `self.current_stage == TrackerState::AllItemFinished` and timeout log log depends on `self.current_stage != TrackerState::AllItemFinished`, there is no chance to output twice